dataset_internal_test.go

```diff
@@ -0,0 +1,64 @@
+package ep
+
+import (
+	"github.com/panoplyio/ep/compare"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestAllMergeOptionResults(t *testing.T) {
```

Comment:
```suggestion
func TestMerge(t *testing.T) {
```
see test convention name [here](https://golang.org/pkg/testing/) (se decided to use Example naming conventions also for tests):
```
The naming convention to declare examples for the package, a function F, a type T and method M on type T are:

func Example() { ... }
func ExampleF() { ... }
func ExampleT() { ... }
func ExampleT_M() { ... }
```